src/Compiler/Microsoft.AspNetCore.Razor.Language/test/DefaultRazorIntermediateNodeLoweringPhaseIntegrationTest.cs

```diff
@@ -525,6 +525,7 @@ public DesignTimeOptionsFeature(bool designTime)
         public void Configure(RazorParserOptionsBuilder options)
         {
             options.SetDesignTime(_designTime);
+            options.UseRoslynTokenizer = true;
```

Comment:
Can we also run razor-toolset-ci pipeline (which tests bunch of existing razor projects) with the roslyn tokenizer enabled?

Reply:
Will do this before we merge the feature branch and address any breaks that come up with dedicated tests at that point.